stable/coredns/README.md

```diff
@@ -79,7 +79,16 @@ The command removes all the Kubernetes components associated with the chart and
 | `extraVolumeMounts`                     | Optional array of volumes to mount inside the CoreDNS container                       | []                                                          |
 | `extraSecrets`                          | Optional array of secrets to mount inside the CoreDNS container                       | []                                                          |
 | `customLabels`                          | Optional labels for Deployment(s), Pod, Service, ServiceMonitor objects               | {}                                                          |
+| `RollingUpdate.maxUnavailable`          | Maximum number of unavailable replicas during rolling update                          | `1`                                                         |
```

Comment:
Sorry, I missed this in my first review, can you make this and the following one lowercase?